app/models/user.rb

```diff
@@ -1,11 +1,21 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
```

Comment:
- [ ] The application should ask for confirmation by email. kindly add confirmable to devise